pkg/mcs/register_test.go

```diff
@@ -0,0 +1,60 @@
+package mcs
+
+import (
+	"testing"
+)
+
+func TestIsAvailable(t *testing.T) {
+	// Test IsAvailable function
+	available = true
+	if !IsAvailable() {
+		t.Error("Expected IsAvailable to return true when available is true")
```

Comment:
Why aren't we using `assert.True(t, IsAvailable())` and instead we use if statements for assertions?